README.md

```diff
@@ -25,18 +25,24 @@ added: REPLACEME
       times. If `true`, all values will be collected in an array. If
       `false`, values for the option are last-wins. **Default:** `false`.
     * `short` {string} A single character alias for the option.
-  * `strict`: {boolean} Should an error be thrown when unknown arguments
+  * `strict` {boolean} Should an error be thrown when unknown arguments
     are encountered, or when arguments are passed that do not match the
     `type` configured in `options`.
     **Default:** `true`.
-  * `allowPositionals`: {boolean} Whether this command accepts positional
+  * `allowPositionals` {boolean} Whether this command accepts positional
     arguments.
     **Default:** `false` if `strict` is `true`, otherwise `true`.
+  * `tokens` {boolean} Return an array with the
```

Comment:
I like `withTokens`, but not a blocker for me. Also something we can discuss in an upstream PR against Node.

Reply:
I do kind of prefer just `tokens`, but don't feel strongly.

In the interests of data, here's what some other parsers in the ecosystem use for their "enable tokens" or "enable locations" options:

- [babel](https://babeljs.io/docs/en/babel-parser#options) uses `tokens: true`
- [flow-parser](https://www.npmjs.com/package/flow-parser) uses `tokens: true`
- [acorn](https://www.npmjs.com/package/acorn) uses `locations: true`
- [espree](https://github.com/eslint/espree#options) uses `loc: true`
- [graphql-js](https://github.com/graphql/graphql-js/blob/47bd8c8897c72d3efc17ecb1599a95cee6bac5e8/src/language/parser.ts#L79) uses `noLocation: false`
- [parse5](https://parse5.js.org/interfaces/parse5.ParserOptions.html#sourceCodeLocationInfo) uses `sourceCodeLocationInfo: true`
- [jsdom](https://github.com/jsdom/jsdom#simple-options) uses `includeNodeLocations: true`

So `jsdom` is the only parser I came across which uses a name which is not just a noun  (except graphql), and it uses `include` as the prefix. Following `jsdom` would be `includeTokens`, which feels a little nicer than `withTokens` to me.

Reply:
I'll resolve this thread and add a link to: https://github.com/pkgjs/parseargs/pull/129#issuecomment-1146441974